ejb3/src/main/java/org/jboss/as/ejb3/subsystem/LegacyCacheFactoryResourceDefinition.java

```diff
@@ -64,10 +64,10 @@ public class LegacyCacheFactoryResourceDefinition extends SimpleResourceDefiniti
     private static final LegacyCacheFactoryRemove REMOVE_HANDLER = new LegacyCacheFactoryRemove(ADD_HANDLER);
 
     LegacyCacheFactoryResourceDefinition() {
-        super(PathElement.pathElement(EJB3SubsystemModel.CACHE),
-                EJB3Extension.getResourceDescriptionResolver(EJB3SubsystemModel.CACHE),
-                ADD_HANDLER, REMOVE_HANDLER,
-                OperationEntry.Flag.RESTART_NONE, OperationEntry.Flag.RESTART_RESOURCE_SERVICES);
+        super(new SimpleResourceDefinition.Parameters(PathElement.pathElement(EJB3SubsystemModel.CACHE), EJB3Extension.getResourceDescriptionResolver(EJB3SubsystemModel.CACHE))
```

Comment:
Nit: The Parameters class does not need to be qualified with its parent class here.